app/mailers/application_mailer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: 'planner.api.mailer@gmail.com'
+  default from: Constants::Shared::EMAIL_ADDRESS
```

Comment:
`Constants::Shared::DEFAULT_EMAIL` gets you more context. Also more flexible way use value from `Rails.configuration.default_sender_email`, for example.

Reply:
Rails.configuration used